CBForest/KeyStore.cc

```diff
@@ -106,14 +106,14 @@ namespace forestdb {
             memcpy(keybuf, key.buf, key.size);
             key.buf = keybuf;
         }
-        fdb_doc doc = {
-            .key = (void*)key.buf,
-            .keylen = key.size,
-            .meta = (void*)meta.buf,
-            .metalen = meta.size,
-            .body  = (void*)body.buf,
-            .bodylen = body.size,
-        };
+        fdb_doc doc;
```

Comment:
For safety this line should be `fdb_doc doc = { }`, to make sure all fields are initialized to zero. (The assignments below should fill in all the fields, but just to make sure, and in case any fields ever got added to the struct in the future.)